### TODO
# description
geometry_msgs/Vector3 force_threshold
---
bool success